runner-core/src/main/java/org/apache/apisix/plugin/runner/server/ApplicationRunner.java

```diff
@@ -123,6 +136,35 @@ public void start(String path) throws Exception {
         }
     }
 
+    private static void manageSocketPermissions(String socketFile) throws IOException {
+        Runtime.getRuntime().exec("chmod 700 " + socketFile);
```

Comment:
@chibenwa In my opinion: we keep to use this way, which should be enough.

As I know, the container environment supports for ACLs is incomplete.

Reply:
Ok I'll drop the ACL part. User can still edit rights / ACL / Group after socket creation if need be...

Sorry for the non-pertinant proposal...